src/Grpc.Net.ClientFactory/Internal/GrpcCallInvokerFactory.cs

```diff
@@ -75,7 +77,7 @@ private CallInvoker CreateInvoker(EntryKey key)
             var httpClientFactoryOptions = _httpClientFactoryOptionsMonitor.Get(name);
             if (httpClientFactoryOptions.HttpClientActions.Count > 0)
```

Comment:
To confirm, the HttpClientActions will just be ignored?

Reply:
I've come up with a semi-hacky way to make them used. Actions now run on a dummy HttpClient and values are then partially applied to gRPC client. It's not perfect, but should cover most scenarios, and a log message provides more info.